### PR TITLE
Foundational admin functionality

### LINKED ISSUE #21: Add admin functionalities to the API
The things needed immediately:
- [x] Broadcast creation
- [x] Broadcast updating
- [x] Broadcast deletion
- [x] Ability to see all waitlist entries
- [x] Ability to invite a waitlist entry
- [x] Ability to delete a waitlist entry
- [x] Mint $run
- [x] Burn $run
- [x] Transfer $run
- [x] Ability to approve a proposal
- [x] Ability to deny a proposal
- [x] Ability to list all users
- [x] Ability to update user

### PR DESCRIPTION
**Pull request by 381e35**

### CODE REVIEW

File: api/governance/views.py [hunk 1]
```diff
@@ -1,15 +1,127 @@
...
 from django_filters.rest_framework import DjangoFilterBackend
 
 from .models import Proposal
+from .permissions import CanManageProposal
 from .serializers import ProposalSerializer, ProposalVoteSerializer
 
 
-class ProposalVoteView(views.APIView):
-    def post(self, request, *args, **kwargs):
+# TODO: Make sure that pending proposals can only be viewed by the proposer
```
Comment (masonchain) [inline api/governance/views.py]: Could not take the time to spin up the API right now, but this test should be the only thing I'd recommend implementing prior to merging.

Should be something like:
```    
def test_cannot_retrieve_unapproved_unowned_proposal(self):
        proposal = Proposal.objects.create(
            title="Test Proposal",
            description="Test proposal",
            proposed_by=self.secondary_user,
            approved=False
        )

        response = self.client.get(
            reverse("proposal-detail", kwargs={"proposal_id": proposal.id}),
            HTTP_AUTHORIZATION=f"Bearer {self.access}"
        )

        self.assertEqual(status.HTTP_404_NOT_FOUND, response.status_code)
```
Might be off with the status code (or does it return an empty queryset?) but should be close.

**Review by masonchain — APPROVED** [review]
Just one in-file TODO to implement a test for proposal retrieving a non-approved and unowned proposal.

Badass to see the architecture scaling so efficiently my man 👍

File: api/governance/views.py (hunk 1 above)
Comment (381e35) [reply api/governance/views.py]: Already covered by:

```
    def test_cannot_update_proposal_for_other_user(self):
        proposal = Proposal.objects.create(
            title="Test Proposal",
            description="Test proposal",
            proposed_by=self.user,
            approved=True
        )

        response = self.client.put(
            reverse("proposal-detail", kwargs={"proposal_id": proposal.id}),
            data={
                "title": "Updated Title",
                "description": "Updated description",
            },
            HTTP_AUTHORIZATION=f"Bearer {self.secondary_access}"
        )

        self.assertEqual(status.HTTP_404_NOT_FOUND, response.status_code)
```

Comment (381e35) [reply api/governance/views.py]: Further resolved by the new structure.

**Review by 381e35 — COMMENTED** [review]
Can't find anything. Please make sure that everything is good and taken care of so that you have everything you need. Believe at this point it is back to moving forwards.

**Pull request merged**